charts/seed-operatingsystemconfig/original/values.yaml

```diff
@@ -16,6 +16,9 @@ kubernetes:
   version: 1.11.2
 worker:
   name: cpu-worker
+  cri:
+#   name: containerd
+    enableContainerD: false # true
```

Comment:
For me this seems to be a duplicated field because `name == containerd` implies that `enableContainerD: true`, right? Can you instead write a helper function in the chart?

Reply:
Agreed, makes sense to remove it in favour of the name?

Reply:
I couldn't find any information about helper function to helm chart.
Do you have an example maybe?

Reply:
The reason i didn't check the string: containerd in the template is because the const is in the code.
"enableContainerD": worker.CRI != nil && worker.CRI.Name == gardencorev1beta1.CRINameContainerD,
What can be done is sending also the available CRI names as parameters to the template and compare the string in the template. 

This condition will remain the same:
  {{- if .Values.worker.cri.name }}

And this condition:
{{- if .Values.worker.cri.enableContainerD }}
will be changed for exmple to:
{{- if  eq .Values.worker.cri.name osc.cri.containerdName

I'm not sure is the best approach. For example enable or disable other CRI can be different conditions.

Reply:
> Agreed, makes sense to remove it in favour of the name?

It does make sense to keep the name. ContainerD is highly coupled with those conditions.

Reply:
Yes, let's remove the `enableContainerD` in the API - what you are referring to with respect to the Helm chart is an implementation detail that should not influence how the API looks like.

Reply:
@rfranzke This is a value relevant only to render the helm chart.
In the OSC spec (API) there is no enableContainerD, only cri.name

Reply:
Sorry, I missed this. Seeing it now. OK, understood. Fine for me. @timuthy ?

Reply:
> What can be done is sending also the available CRI names as parameters to the template and compare the string in the template.

I actually really like this idea :)

Reply:
Ok i changed it